src/session.ts

```diff
@@ -52,6 +49,15 @@ export const enum types {
   ReadWrite = 'readwrite',
 }
 
+export type GetSessionMetadataCallback = ResourceCallback<
```

Comment:
`getMetadata` returns only one value=> the session metadata itself (`ISession`)
[Ref1](https://github.com/laljikanjareeya/nodejs-spanner/blob/4d3d64962a1d4c7fafd2b8610209c7aed6b0585b/proto/spanner.d.ts#L228), [Ref2](https://github.com/laljikanjareeya/nodejs-spanner/blob/4d3d64962a1d4c7fafd2b8610209c7aed6b0585b/src/v1/spanner_client.d.ts#L132-L134)
please use
```suggestion
export type GetSessionMetadataCallback = NormalCallback<
  spanner_client.spanner.v1.ISession
>;
```